scripts/rebase-fork.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+if [[ -z $1 ]]; then
+    echo "Please specify the pull request number you want to rebase to a local branch."
+    echo "Usage: ./scripts/rebase-fork.sh <pr number>"
+    echo ""
+    echo "Example: ./scripts/rebase-fork.sh 557"
+    exit 1
+fi
+
+PR_NUMBER=$1
+
+declare -a req_tools=("gh" "git" "jq")
+for tool in "${req_tools[@]}"; do
+  if ! command -v "$tool" > /dev/null; then
+    echo "It looks like '${tool}' is not installed; please install it and run this script again."
+    exit 1
+  fi
+done
+
+# Check if the PR specified is a valid number
+re='^[0-9]+$'
+if ! [[ $PR_NUMBER =~ $re ]] ; then
+   echo "The PR you specify must be a valid integer number." >&2; exit 1
+fi
+
+# Checkout the fork PR locally
+gh pr checkout $PR_NUMBER
```

Comment:
I tried entering an invalid PR number and the script does not fail. I think we should also check for this case:

```
awinkler in ~/dev/go-tfe on rebase-script ● λ ./scripts/rebase-fork.sh 998234324
GraphQL: Could not resolve to a PullRequest with the number of 998234324. (repository.pullRequest)
GraphQL: Could not resolve to a PullRequest with the number of 998234324. (repository.pullRequest)
GraphQL: Could not resolve to a PullRequest with the number of 998234324. (repository.pullRequest)
GraphQL: Could not resolve to a PullRequest with the number of 998234324. (repository.pullRequest)
fatal: 'local/' is not a valid branch name
[rebase-script 4614d73] Rebased  onto a local branch
fatal: invalid refspec 'local/'
Warning: 5 uncommitted changes
? Where should we push the 'rebase-script' branch?  [Use arrows to move, type to filter]
> hashicorp/go-tfe
  Create a fork of hashicorp/go-tfe
  Skip pushing the branch
  Cancel
```